SMLHelper/Patchers/PingTypePatcher.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Harmony;
+using SMLHelper.V2.Handlers;
+using SMLHelper.V2.Utility;
+
+namespace SMLHelper.V2.Patchers
+{
+    internal class PingTypePatcher
+    {
+        private const string TechTypeEnumName = "PingType";
+        internal static readonly int startingIndex = 10;
```

Comment:
Should start over the max value of `PingType` which we can use as a constant.
```CS
internal const int startingIndex = (int)PingType.Sunbeam + 1;
```

Reply:
PingType doesn't seem to rely on consecutive numbers so we could start from e.g. 1000 to give some breathing room to below zero incase they ever add a new ping type

Reply:
That works too.